index.js

```diff
@@ -310,19 +324,74 @@ export function buildJsx(tree, options = {}) {
       }
 
       if (automatic) {
-        if (children.length > 1) {
+        parameters.push(props || {type: 'ObjectExpression', properties: []})
+
+        if (key) {
+          parameters.push(key)
+        } else if (options.development) {
+          parameters.push({
+            type: 'UnaryExpression',
+            operator: 'void',
+            prefix: true,
+            argument: {type: 'Literal', value: 0}
+          })
+        }
+
+        const isStaticChildren = children.length > 1
+        if (options.development) {
+          imports.jsxDEV = true
+          callee = {
+            type: 'Identifier',
+            name: '_jsxDEV'
+          }
+          parameters.push({type: 'Literal', value: isStaticChildren})
+          if (options.filePath) {
```

Comment:
I figured it’s probably useless to add positional information if the file path is missing.